lib/language_pack/base.rb

```diff
@@ -206,11 +206,11 @@ def log(*args)
   def setup_language_pack_environment
   end
 
-  def add_to_profiled(string)
+  def add_to_profiled(string, filename = "ruby.sh", mode = "a")
```

Comment:
This is a clean refactor. I like this approach.

My strong preference to args is to use named args `filename: "ruby.sh"`. I was going to try to point at a "convention" list I'm keeping at https://github.com/heroku/buildpacks-ruby#coding-conventions but it looks like that didn't make the cut. I'll need to update that.

Can you switch over to named args?

Reply:
sure